src/partials/teachers-section.html

```diff
@@ -0,0 +1,68 @@
+<section class="teachers" id="teachers">
+    <div class="container">
+        <h2 class="section-title">Meet our teachers</h2>
```

Comment:
Wrap word teachers into span and add class "teachers"

Reply:
done